js/downloads.js

```diff
@@ -15,14 +15,14 @@ var sources = {pretty: "Source Code", tag: "sources"};
 var hadoopFree = {pretty: "Pre-build with user-provided Apache Hadoop", tag: "without-hadoop"};
 var hadoop2p6 = {pretty: "Pre-built for Apache Hadoop 2.6", tag: "hadoop2.6"};
 var hadoop2p7 = {pretty: "Pre-built for Apache Hadoop 2.7 and later", tag: "hadoop2.7"};
-var scala2p12_hadoopFree = {pretty: "Pre-build with Scala 2.12 and user-provided Apache Hadoop", tag: "without-hadoop-scala-2.12"};
+var scala2p11_hadoopFree = {pretty: "Pre-build with Scala 2.11 and user-provided Apache Hadoop", tag: "without-hadoop-scala-2.11"};
```

Comment:
@srowen do we still need it? In the past we provided a scala 2.12 build for users to try out. Now we have moved to scala 2.12, do we need to leave a special scala 2.11 build?